src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -133,6 +133,13 @@ public static class TreeExtensions
         /// <param name="numberOfLeaves">The maximum number of leaves per decision tree.</param>
         /// <param name="minimumExampleCountPerLeaf">The minimal number of data points required to form a new tree leaf.</param>
         /// <param name="learningRate">The learning rate.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[FastTreeBinaryClassification](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/FastTree.cs)]
```

Comment:
>FastTreeBinaryClassification [](start = 26, length = 28)

Please have the names match the destination file.

Reply:
Thank you for pointing out!

---
In reply to: [275908194](https://github.com/dotnet/machinelearning/pull/3338#discussion_r275908194) [](ancestors = 275908194)